### ISSUE TITLE
Remove pljson => json aliases

### ISSUE THREAD

**Issue by jsumners** [pljson/pljson#54]
This issue is to be addressed when we are ready to create a 3.0.0 release. Such a release should not happen prior to 2016-10-01 in order to give people time to adjust to 2.0.0.

**Comment by dsnz** [comment]
whatever for a 3.0.0 release we cannot remove the aliases
there are old users with code using original names starting with json_
if they update (and they should) their apps will break

a related issue here is how many users are actively using pljson and what version
(Pulse indicates ~500 unique visitors last month, how many could be real users ?)

**Comment by jsumners** [comment]
Right. That's what I'm thinking the 2.x.x release will be for -- transitional compatibility. Eventually they will need to be removed so that there is less to maintain. The target date for 3.0.0 doesn't have be October. It could be pushed out to whenever.
